standard/currency/currency.lua

```diff
@@ -92,4 +92,28 @@ function Currency.formatMoney(value)
 	return LANG:formatNum(integer) .. string.format('%.2f', decimal):sub(2)
 end
 
+function Currency.getExchangeRate(props)
+	if not props then
+		error('No props passed to "Currency.getExchangeRate"')
+	end
+	local setVariables = Logic.readBool(props.setVariables)
+	local currencyRate = tonumber(props.currencyRate)
+	local currency = props.currency:upper()
+	if not currencyRate then
+		if String.isEmpty(props.currency) then
+			error('No currency passed to "Currency.getExchangeRate"')
+		end
+		if not props.date:match('%d%d%d%d%-%d%d%-%d%d') then
+			error('Invalid date passed to "Currency.getExchangeRate"')
+		end
```

Comment:
Does `currencyexchange` support unix time input in teh date? If so, probably best not to ruin that

Reply:
idk
but every infobox extension as well as pp check for valid dates anyways (both using `:match('%d%d%d%d%-%d%d%-%d%d')`)

Reply:
Ok